third_party/prometheus/Makefile

```diff
@@ -1,19 +1,26 @@
 VERSION=2.37.1
+SHA256_linux_x86_64=753f66437597cf52ada98c2f459aa8c03745475c249c9f2b40ac7b3919131ba6
```

Comment:
For the record, I confirmed the checksums against https://github.com/prometheus/prometheus/releases/tag/v2.37.1

Reply:
I updated and rechecked the checksums.